### DISCUSSION TITLE
warnings in the new version of Parcels

### DISCUSSION THREAD

**Discussion by RomanIMMSP** [Parcels-code/Parcels#1272] [Q&A] [answered]
Dear OceanParcels team,

I just installed the last version of Parcels and ran the code that works well in the previous version.
As you may understand, I received some warnings (see screen below).
My question is about the importance of these warnings. Why do they appear and how to avoid them?
![image](https://user-images.githubusercontent.com/101412113/210411062-3b0b8f0d-b317-448b-885b-e071060c2904.png)

Thanks in advance for explanation.

**Comment by erikvansebille** [comment]
Thanks for reporting, @RomanIMMSP. I have not seen these warnings myself. Which version of Python are you using? Seems to be some internal Python warning..

**Reply by RomanIMMSP** [reply]
I have installed the last version of Anaconda with Python 3.9.
Probably I need to install Miniconda instead of Anaconda?

**Reply by michaeldenes** [reply, marked as answer]
I believe this is fixed in #1400, thrown by numpy > 1.20.3.
